test/MOI_Wrapper.jl

```diff
@@ -0,0 +1,182 @@
+using Xpress, MathOptInterface, Test
+
+const MOI = MathOptInterface
+const MOIT = MathOptInterface.Test
+
+const MOIT = MOI.Test
+const MOIU = MOI.Utilities
+
+const OPTIMIZER = Xpress.Optimizer()
+const OPTIMIZER_2 = Xpress.Optimizer()
+const BRIDGED_OPTIMIZER = MOI.Bridges.full_bridge_optimizer(
+    Xpress.Optimizer(), Float64)
+
+const CONFIG = MOIT.TestConfig()
+
+@testset "SolverName" begin
+    @test MOI.get(OPTIMIZER, MOI.SolverName()) == "Xpress"
+end
+
+@testset "supports_default_copy_to" begin
+    @test MOIU.supports_default_copy_to(OPTIMIZER, true)
+end
+
+@testset "Unit Tests Constraints" begin
+    MOIT.basic_constraint_tests(OPTIMIZER, CONFIG, delete = false) # TODO: remove `delete = false`
+end
+
+@testset "Unit Tests" begin
+    MOIT.unittest(BRIDGED_OPTIMIZER, CONFIG, #= excludes =# [
+        # "add_variable",
+        # "add_variables",
+        # "delete_nonnegative_variables",
+        "delete_soc_variables", # TODO: fix errors
+        # "delete_variable",
+        # "delete_variables",
+        # "feasibility_sense",
+        # "get_objective_function",
+        # "getconstraint",
+        # "getvariable",
+        # "max_sense",
+        # "min_sense",
+        # "number_threads",
+        # "raw_status_string",
+        # "silent",
+        # "solve_affine_deletion_edge_cases",
+        # "solve_affine_equalto",
+        # "solve_affine_greaterthan",
+        # "solve_affine_interval",
+        # "solve_affine_lessthan",
+        # "solve_blank_obj",
+        # "solve_constant_obj",
+        # "solve_duplicate_terms_obj",
+        # "solve_duplicate_terms_scalar_affine",
+        # "solve_duplicate_terms_vector_affine",
+        # "solve_integer_edge_cases",
+        # "solve_objbound_edge_cases",
+        "solve_qcp_edge_cases", # TODO: fix errors (?). This might be tested below.
+        "solve_qp_edge_cases", # TODO: fix errors (?). This might be tested below.
+        # "solve_result_index",
+        # "solve_single_variable_dual_max",
+        # "solve_single_variable_dual_min",
+        # "solve_singlevariable_obj",
+        # "solve_time",
+        # "solve_unbounded_model",
+        # "solve_with_lowerbound",
+        # "solve_with_upperbound",
+        # "solve_zero_one_with_bounds_1",
+        # "solve_zero_one_with_bounds_2",
+        # "solve_zero_one_with_bounds_3",
+        # "solver_name",
+        # "time_limit_sec",
+        # "update_dimension_nonnegative_variables",
+        # "variablenames",
+       ],
+    )
+    MOIT.modificationtest(BRIDGED_OPTIMIZER, CONFIG)
+end
+
+SIMPLE_CONFIG = MOIT.TestConfig(basis = false, infeas_certificates=false)
+
+@testset "Linear tests" begin
+    @testset "Default Solver"  begin
+        MOIT.contlineartest(BRIDGED_OPTIMIZER, MOIT.TestConfig(
+        basis = false, # TODO change to true
+        infeas_certificates=false # TODO remove this

```

Comment:
shouldn't it be true?
and remove the TODOs?

Reply:
Resolved in https://github.com/JuliaOpt/Xpress.jl/pull/44/commits/9e4e19d29656484a5b181fff4256773c397417e1. I'll wait till tests pass on Travis to resolve this comment.

Reply:
sure! resolve when it passes.